ui.frontend/src/constants.js

```diff
@@ -349,6 +349,18 @@ export const Constants = {
      * Prefix path for all AF HTTP APIs.
      * @type {string}
      */
-    API_PATH_PREFIX : "/adobe/forms/af"
+    API_PATH_PREFIX : "/adobe/forms/af",
+
+    /**
+     * Html input type text
+     * @type {string}
+     */
+    HTML_INPUT_TYPE_TEXT : "text",
```

Comment:
don't add constants here, you can keep them in view

Reply:
removed from constants.js and moved to view.